app/scripts/views/complete_reset_password.js

```diff
@@ -25,6 +28,9 @@ function (Cocktail, BaseView, FormView, Template, PasswordMixin,
     initialize: function (options) {
       options = options || {};
 
+      // TODO - pass this in!
```

Comment:
Yes, yes we should. I'm not sure it's strictly necessary, but my gut tells me the right thing to do is pass the model in. The reason I haven't done it so far creating and passing in an object to only one view is something we haven't done before, and I haven't given any thought to. I don't think it's difficult, I just haven't sat down and thought in depth about it.